go/test/endtoend/vtgate/queries/subquery/subquery_test.go

```diff
@@ -58,8 +58,6 @@ func TestSubqueriesHasValues(t *testing.T) {
 }
 
 func TestNotINQueries(t *testing.T) {
-	utils.SkipIfBinaryIsBelowVersion(t, 19, "vtgate")
-
```

Comment:
Oh you still need to skip the test if the vtgate version in less than v18 if it doesn't work with v18 vtgate